lib/harvesting/errors.rb

```diff
@@ -4,4 +4,7 @@ class AuthenticationError < StandardError
 
   class UnprocessableRequest < StandardError
   end
+
+  class RequestNotFound < StandardError
+  end
```

Comment:
@JuannFerrari And then here we could include the string we want to use: 

```suggestion
    def initialize(uri)
      super( "The page you were looking for may have been moved or the address misspelled: #{uri}")
    end
  end
```